src/scripts.js

```diff
@@ -46,21 +46,23 @@ var userStairsThisWeek = document.getElementById('userStairsThisWeek');
 var userMinutesThisWeek = document.getElementById('userMinutesThisWeek');
 var bestUserSteps = document.getElementById('bestUserSteps');
 var streakList = document.getElementById('streakList');
-var streakListMinutes = document.getElementById('streakListMinutes')
+var streakListMinutes = document.getElementById('streakListMinutes');
+
+window.onload = startApp(); //Starts Here (global variables currentInformation = {})
```

Comment:
good call moving this to the top

Reply:
Yeah this makes a lot more sense to have it up top.